src/__mock__/itemCounter.test.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import { countShows } from '../modules/counterItem.js';
+
+test('properly counts total item on the page', () => {
+  document.body.innerHTML = `
+    <div class="card" id="21"></div>
+    <div class="card" id="22"></div>
+    <div class="card" id="23"></div>
+    <div class="card" id="24"></div>
+    <div class="card" id="25"></div>
+    <div class="card" id="26"></div>
+    <div class="card" id="142270"></div>
+    <div class="card" id="151048"></div>
+    <div class="card" id="151645"></div>
+    <div class="card" id="153120"></div>
```

Comment:
- [ ] These counter functions need to be covered with unit tests using Jest.  Great job so far 👏🏻 however you should think at least about the scenario in which there are 0 items on the page (counter function should return 0). kindly add this 👍🏻

Reply:
Done. ✅